src/linalg.jl

```diff
@@ -31,15 +31,15 @@ LinearAlgebra.mul!(
     adjB::Adjoint{T,<:BlockedSparse{T}},
     α::Number,
     β::Number,
-) where {T} = mul!(C, A, adjB.parent.cscmat', α, β)
+) where {T} = mul!(C, A, adjoint(adjB.parent.cscmat), α, β)
```

Comment:
was this just because of the linter complaining?

Reply:
Yes.  I think they may have even fixed the issue in the linter used with VS Code now.  Your call on whether you want to revert this change or not.

Reply:
I think readability is about the same either way, so I can't be bothered to change it back. :smile: